### PR TITLE
Update superset kfdef in osc-cl2.

### PR DESCRIPTION
**Pull request by HumairAK**
This change will point superset kfdef deployment to the superset
manifests supplied by odh version 1.1.2. This change allows us to manage
certain secrets ourselves instead of relying on the odh operator. As
such these seecrets have been also added to the kfdef superset app.

ODH upgrading to ODH v1.1.2 will also update Superset version to 1.3.2.

Related to: https://github.com/open-services-group/devsecops/issues/9

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
We are removing the `SUPERSET_IMAGE` parameter because the it will default to latest correct? If that is the case shouldn't we lock its version at `v1.3.2`?

**Comment by HumairAK** [comment]
@Gregory-Pereira Good question, ODH actually hardcoded their superset image since v1.1.1. So we can't update it via this parameter going forward.

**Comment by Gregory-Pereira** [comment]
> @Gregory-Pereira Good question, ODH actually hardcoded their superset image since v1.1.1. So we can't update it via this parameter going forward.
> 
> If you're curious: Our fork: https://github.com/operate-first/odh-manifests/blob/master/superset/base/kustomization.yaml#L119 Upstream: https://github.com/opendatahub-io/odh-manifests/blob/master/superset/base/kustomization.yaml#L119

Thanks for explaining.
/lgtm

File: kfdefs/overlays/osc/osc-cl2/superset/kfdef.yaml [hunk 1]
```diff
@@ -11,13 +11,15 @@ spec:
       name: odh-common
     - kustomizeConfig:
         parameters:
```
Comment (4n4nd) [inline kfdefs/overlays/osc/osc-cl2/superset/kfdef.yaml:13]: is this still a valid yaml?

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**